massa-node/src/main.rs

```diff
@@ -649,6 +657,17 @@ async fn main(args: Args) -> anyhow::Result<()> {
         .with(tracing_layer)
         .init();
 
+    let x = match args.boot {
+        None => {
```

Comment:
You can use is_some : https://doc.rust-lang.org/std/option/enum.Option.html#method.is_some to perform this conversion